src/Service/MergeRequestService.php

```diff
@@ -53,4 +67,24 @@ protected function updateReviewStatus(MergeRequest $mergeRequest): void
 
         $review->setStatus(Status::CLOSED);
     }
+
+    private function processNewMergeRequest(MergeRequest $mergeRequest)
+    {
+        $requestedScope = $this->findScopeFromDescription($mergeRequest);
+        if ($requestedScope !== null) {
+            $this->gitlabService->notifyAboutInitialReviewRequest($mergeRequest, $requestedScope);
```

Comment:
The current implementation will only post a `Hang on, I'm searching for reviewers for this merge request %s` comment, but will not assign reviewers.

To assign reviewers, we would need to add a method to ReviewService, something along the lines of:
```

    public function createFromMergeRequest(MergeRequest $mergeRequest, Scope $scope): Review
    {
        $review = $this->findByMergeRequest($mergeRequest);
        if ($review === null) {
            $review = new Review();
        }

        $review->setProject($mergeRequest->getProject());
        $review->setMergeRequest($mergeRequest);
        $review->setScope($scope->getName());

        $this->entityManager->persist($review);
        $this->entityManager->flush();

        return $review;
    }
```

and then invoke it here like so

```
if ($requestedScope !== null) {
            $review = $this->reviewService->createFromMergeRequest($mergeRequest, $requestedScope);
            $this->reviewService->notifyAboutReadyReviewsOnComment($review);
        }
```